index.js

```diff
@@ -27,10 +31,17 @@ function updateSpecFromPackage() {
     spec.info.license = { name: packageInfo.license };
   }
 
-  spec.info.description = '[Specification JSON](/api-spec)';
+  if (packageInfo.baseUrlPath) {
+    spec.info.description = '[Specification JSON]('+packageInfo.baseUrlPath +'/api-spec) , base url : ' + packageInfo.baseUrlPath;
```

Comment:
please use same formatting style everytwhre: there should be spaces around `+` here and below